docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+Now look at the standard ``release_beta`` flow defined in the universal ``cumulusci.yml`` file:
+
+.. code-block:: yaml
+
+   release_beta:
+        description: Upload and release a beta version of the metadata currently in packaging
+        steps:
+            1:
+                task: upload_beta
+                options:
+                    name: Automated beta release
+            2:
+                task: github_release
+                options:
+                    version: ^^upload_beta.version_number
+            3:
+                task: github_release_notes
+                ignore_failure: True  # Attempt to generate release notes but don't fail build
+                options:
+                    link_pr: True
+                    publish: True
+                    tag: ^^github_release.tag_name
+                    include_empty: True
+                    version_id: ^^upload_beta.version_id
+            4:
+                task: github_master_to_feature
+
+This flow shows how subsequent tasks can reference the return values of a prior task. In this case, the ``github_release`` task uses the ``version_numer`` set by the ``upload_beta`` task as an option value with the ``^^upload_beta.version_number`` syntax. Similarly, the ``github_release_notes`` task uses the ``version_id`` set by the ``upload_beta`` task as an option value with the ``^^upload_beta.version_id`` syntax.
+
+
+
+Flow Configurations
+-------------------
+
+Flow configurations are defined under the ``flows`` section of your ``cumulusci.yml`` file. Each flow configuration defines a flow which can be run using the ``cci flow run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
+
+Add a Custom Flow
+^^^^^^^^^^^^^^^^^
+
+To define a new flow for your project, simply add the name of the new flow under the ``flows`` section of your ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    greet_and_sleep:
+        group: projectName
+        description: Greets the user and then sleeps for 5 seconds.
+        steps:
+            1:
+                task: command
+                options:
+                    command: echo 'Hello there!' 
+            2:
+                task: util_sleep
+
+This is a flow comprised of two tasks: ``command`` greets the user by echoing a string, and ``util_sleep`` then tells CumulusCI to sleep for five seconds. You can reference how the flows are defined in the `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file.
+
+
+Add a Flow Step
+^^^^^^^^^^^^^^^
+
+If you want to add a step to a flow, use ``cci flow info <name>`` first to see the existing steps.
+
+    Example: ``dev_org``
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
```

Comment:
Is "unmanaged" or "unpackaged" preferred here?

Reply:
@Br4nd0R Please confirm.

Reply:
Unmanaged is what we want.